src/app/sort/sort.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SortComponent } from './sort.component';
+
+describe('SortComponent', () => {
```

Comment:
If test files are created automatically make sure the test suite runs ok.

(I did not check that, so maybe is working)